backend/src/ml_space_lambda/initial_app_config/lambda_function.py

```diff
@@ -44,5 +45,30 @@ def lambda_handler(event, context):
     ]
 
     app_configuration_dao.update(config)
+    update_dynamic_roles()
 
     generate_html_response(200, "Successfully updated app config")
+
+
+def update_dynamic_roles():
```

Comment:
Do we have to add some check to see if we are managing the IAM roles for them? I think we have the var `export const MANAGE_IAM_ROLES = true;`

Reply:
I just added that env var to the lambda and put in a check to only attempt this if `MANAGE_IAM_ROLES` is true.